Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -257,14 +289,116 @@ public bool TryUpdateQueryBufferForLayerMask(LayerMask layerMask, Vector3 pointe
 
                 for (int i = 0; i < numColliders; i++)
                 {
-                    grabbable = queryBuffer[i].GetComponent<NearInteractionGrabbable>();
+                    Collider collider = queryBuffer[i];
+                    grabbable = collider.GetComponent<NearInteractionGrabbable>();
+                    if (grabbable != null)
+                    {
+                        if (ignoreCollidersNotInFOV)
+                        {
+                            if (!isInFOVCone(collider))
```

Comment:
Simplify if statments to something maybe like below?

if (grabbable != null)
{
if (ignoreCollidersNotInFOV && !isInFOVCone(collider))
{
/// comments
grabbable = null;
}
else
{
return true;
}
}

Reply:
Yeah I know we used to have that, but actually I like keeping in separate since it makes it easier to profile. I'll add a comment explaining why it is separate. It also makes it a bit more readable IMO, even though it's a little more lines.